### PR TITLE
Enabling CI

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/639

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by pabelanger** [comment]
recheck

File: zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+- project:
```
Comment (pabelanger) [inline zuul.yaml:2]: lets use ansible-collections-community-yang-units template here

**Comment by pabelanger** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
Closing and opening https://github.com/ansible-collections/community.yang/pull/13

**Pull request closed without merging**